packs/data/feats.db/trap-finder.json

```diff
@@ -25,65 +25,20 @@
             {
                 "domain": "all",
                 "key": "RollOption",
-                "label": "Trap Finder (detecting or defending against traps)",
-                "option": "trap-finder",
+                "label": "PF2E.SpecificRule.TrapFinder.Label",
+                "option": "origin:trait:trap",
```

Comment:
?

Reply:
I guess I could make the predicates an `or` and keep that as `trap-finder`.